packages/tradershub/src/components/DemoRealSwitcher/DemoRealSwitcher.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
```

Comment:
let's make this a generic components which accepts an array of different states and put it in the library and import it here for the demo/real switcher so once the quill ListOption is there, we can replace it

Reply:
We can do that but sofar the only part of the app that uses this dropdown is the main Tradershub page (from my knowledge). Plus the stylings, the colors and the behaviors might be different. Maybe in a future PR so I can see if there's any other shared similarities with other dropdowns.